src/openvic-simulation/pop/Culture.cpp

```diff
@@ -62,7 +65,12 @@ bool CultureManager::add_culture(
 		Logger::error("Invalid culture identifier - empty!");
 		return false;
 	}
-	return cultures.add_item({ identifier, colour, group, std::move(first_names), std::move(last_names) });
+
+	// TODO - check radicalism range
```

Comment:
I've been thinking about how to handle fixed points with limited ranges, we could make a template type that takes the min and max as arguments and automatically clamps it every time any change is made. This will be very useful for stuff like militancy and conciousness, and anything that can't be negative.